apps/ledger-live-mobile/src/families/evm/StakingDrawer/EvmStakingDrawerBody.tsx

```diff
@@ -34,7 +35,10 @@ export function EvmStakingDrawerBody({ providers, accountId, onClose }: Props) {
           navigation.navigate(ScreenName.PlatformApp, {
             platform: manifest.id,
             name: manifest.name,
-            accountId,
+            accountId:
+              manifest?.params && "dappUrl" in manifest.params
+                ? getWalletApiIdFromAccountId(accountId)
+                : accountId,
```

Comment:
Same as desktop, shouldn't we reverse this condition and only send the LL account id for dapp browser v3 apps ?
```suggestion
              manifest?.dapp
                ? accountId
                : getWalletApiIdFromAccountId(accountId),
```